internal/pkg/cli/validate.go

```diff
@@ -699,6 +704,35 @@ func validatePubSubName(name string) error {
 	return nil
 }
 
+func validateTopicsExist(mft interface{}, topicARNs []string, app, env string) error {
+	type subscriptions interface {
+		Subscriptions() []manifest.TopicSubscription
+	}
```

Comment:
is it possible to move the interface outside of the function and get rid of the error?
```suggestion
type subscriberManifest interface {
	Subscriptions() []manifest.TopicSubscription
}

func validateTopicsExist(mft subscriberManifest, topicARNs []string, app, env string) error {
	
```